openapi/clusters_mgmt/v1/openapi.json

```diff
@@ -15986,7 +15986,7 @@
             "$ref": "#/components/schemas/DeleteProtection"
           },
           "disable_user_workload_monitoring": {
-            "description": "Indicates whether the User workload monitoring is enabled or not\nIt is enabled by default",
+            "description": "Indicates whether the User workload monitoring is enabled or not\nIt is enabled by default. This field is deprecated for Hosted Control Plane clusters and will be removed",
```

Comment:
Wouldn't the place to edit this be in ocm api model corresponding model file, which is the source of truth, and then propagate the change all the way here?